server/controllers/notifications.js

```diff
@@ -0,0 +1,98 @@
+const Notification = require("../models/Notification");
+const asyncHandler = require("express-async-handler");
+
+// @desc    Get all Notifications
+// @route   GET /notifications
+// @access  Private
+exports.getNotifications = asyncHandler(async (req, res) => {
+  const { id } = req.user;
+  const userNotification = await Notification.findOne({ receiver: id });
+  if (userNotification && userNotification.notifications.length > 0) {
```

Comment:
I did the if statement because for now there is possible users don't have the notification which could cause an issue if I try to access the property notifications.